scripts/updatePackageHeader.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+
+const getBuildData = require('./getBuildData');
+
+const nextVersion = process.argv[2];
+
+if (!nextVersion) {
+  throw new Error('This script requires a version number to be provided');
+}
+
+// Update the SCSS Package Header
+const SCSSFilePath = path.resolve('scss', 'nes.scss');
+let SCSSFile = fs.readFileSync(SCSSFilePath, 'utf8');
+
+SCSSFile = SCSSFile.replace(/^ {2}Version: development/m, `  Version: ${nextVersion}`);
```

Comment:
`/^ {2}Version/` is exactly the same as `/^  Version/` is there a reason you chose the first way? I understand using a quantifier if you want to save space and want to match a lot of a character like `/^ {8}/` but two spaces is shorter and clearer than ` {2}` especially for people who may be less familiar with regex.

Reply:
Ahh, I didn't actually try it with ESLint running and didn't realize they had a rule for that. Makes total sense 👍